src_c/display.c

```diff
@@ -895,6 +901,14 @@ pg_set_mode(PyObject *self, PyObject *arg, PyObject *kwds)
     state->using_gl = (flags & PGS_OPENGL) != 0;
     state->scaled_gl = state->using_gl && (flags & PGS_SCALED) != 0;
 
+    if (state->scaled_gl){
+        PyErr_WarnEx(PyExc_FutureWarning, "SCALED|OPENGL is experimental and subject to change" ,1);
```

Comment:
Missing check of return code from `PyErr_WarnEx`

Reply:
What exactly does that return value mean? https://docs.python.org/3/c-api/exceptions.html#c.PyErr_WarnEx

Reply:
Never mind I saw your comment in the other thread